frontend/src/concepts/pipelines/content/PipelineJobReferenceName.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Text, TextVariants } from '@patternfly/react-core';
+import { getPipelineJobExecutionCount } from '~/concepts/pipelines/content/tables/utils';
+import { PipelineCoreResourceKF } from '~/concepts/pipelines/kfTypes';
+import useJobRelatedInformation from '~/concepts/pipelines/content/tables/pipelineRun/useJobRelatedInformation';
+
+type PipelineJobReferenceNameProps = {
+  resource: PipelineCoreResourceKF;
+};
+
+const PipelineJobReferenceName: React.FC<PipelineJobReferenceNameProps> = ({ resource }) => {
+  const { getJobInformation } = useJobRelatedInformation();
+  const { data, loading } = getJobInformation(resource);
+
+  return (
+    <>
+      {loading ? (
+        'loading...'
+      ) : data ? (
+        <Text component={TextVariants.p} className="pf-u-pb-sm">
+          {`Run ${getPipelineJobExecutionCount(resource.name)} of ${data?.name}`}
```

Comment:
```suggestion
          Run {getPipelineJobExecutionCount(resource.name)} of {data?.name}
```
Is there a reason to not just. use JSX?